### PR TITLE
added reference to  Saving/Resuming Study with RDB Backend

### PR DESCRIPTION
**Pull request by Hiroyuki-01**
## Motivation

-a subtask of Mention each tutorial page in API pages at least once #2940

## Description of the changes

add a link to rdb tutorial.

### CODE REVIEW

File: optuna/study/study.py [hunk 1]
```diff
@@ -1140,6 +1140,7 @@ def objective(trial):
 
     See also:
         :func:`optuna.create_study` is an alias of :func:`optuna.study.create_study`.
+        The :ref:`rdb` tutorial provides concrete examples to facilitate understanding.
```
Comment (nzw0301) [inline optuna/study/study.py:1143]: Nice!  How about this?
```suggestion

    See also:
        The :ref:`rdb` tutorial provides concrete examples to save and resume optimization using RDB.
```


1. The existing `see also` is not related to the new sentence, so could you separate them>
2. Explain a little bit the tutorial.

**Review by nzw0301 — CHANGES_REQUESTED** [review]

File: optuna/study/study.py (hunk 1 above)
Comment (Hiroyuki-01) [reply optuna/study/study.py:1143]: I'm sorry, I'm not going to be able to finish in time....

Comment (nzw0301) [reply optuna/study/study.py:1143]: If you don't mind, I'll apply this suggestion :)

File: optuna/study/study.py [hunk 2]
```diff
@@ -1141,6 +1141,9 @@ def objective(trial):
     See also:
         :func:`optuna.create_study` is an alias of :func:`optuna.study.create_study`.
 
+    See also:
+        The :ref:`rdb` tutorial provides concrete examples to save and resume optimization using RDB.
```
Comment (nzw0301) [inline optuna/study/study.py:1145]: Could you apply the following change to fix the `flake8` error?
```suggestion
        The :ref:`rdb` tutorial provides concrete examples to save and resume optimization using
        RDB.
```

**Review by nzw0301 — CHANGES_REQUESTED** [review]
Thank you for your update. Could you take a look at my comment?

**Comment by hvy** [comment]
Thanks a lot for the PR!
LGTM except for the comment from @nzw0301 . 
@nzw0301, could you merge this PR once the changes looks good to you as well?

**Comment by Hiroyuki-01** [comment]
There seems to be some error in the CI.
I don't know what to do.
If there is anything I can do, please let me know.

**Review by nzw0301 — APPROVED** [review]
Thanks!

**Pull request merged**